Xsens stm32 cube ide spi

ineelb opened the pull request:
Added STM32CubeIDE SPI project

A lot of users use STM32CubeIDE for embedded applications, so it is important to have an SPI code/project that is compatible with STM32CubeIDE.

xsenssupport committed:
update readme files to markdown format.

ineelb committed:
Merge branch 'Xsens_STM32CubeIDE_SPI' of https://github.com/ineelb/Xsens_Embedded_Example into Xsens_STM32CubeIDE_SPI

xsenssupport commented:
Thanks for your contribution!
I noticed that this PR includes build output files (e.g., .o, .d). These should not be part of the repository.

Could you please:
- Add a .gitignore file to exclude generated binaries and build artifacts, maybe the whole `Debug` folder
- Remove the already committed build files from the PR.
- Place any images into a dedicated subfolder called images/.
- Add a README.md file with clear instructions on how to set up and use the project. You can link to the images in the images/ folder within the README if needed.

Once you update your branch, the PR will automatically reflect the changes. Thanks!

ineelb committed:
Removed the .o, .d built files and added .gitignore

zhouf345 commented:
I noticed that .settings/, .project, .cproject, and .mxproject are already committed in this PR. While .gitignore now excludes .settings/, these files remain in the repo history. It would be cleaner to remove them from the repository.

zhouf345 commented:
It would be helpful to add a README so that customers can get started quickly.

ineelb committed:
Removed .cproject, .mxproject.
Added README file which contains information about how to use this SPI library.

zhouf345 commented:
This looks good to merge.
<img width="1038" height="837" alt="image" src="https://github.com/user-attachments/assets/12b2d66c-b170-4479-ba67-8cd61996906c" />
